Formula/gusto-lima.rb

```diff
@@ -0,0 +1,70 @@
+class GustoLima < Formula
+  desc "Gusto's opinionated colima profile"
+  homepage "https://github.com/abiosoft/colima/blob/main/README.md"
+  url "https://github.com/abiosoft/colima.git",
+      tag:      "v0.1.0"
+  license "MIT"
+  head "https://github.com/abiosoft/colima.git", branch: "main"
+
+  depends_on "colima"
+  depends_on "docker"
+  depends_on "docker-buildx"
+  depends_on "docker-compose"
+  depends_on "docker-credential-helper"
+  depends_on "docker-credential-helper-ecr"
+
+  def install
+
+    bin.install_symlink "/opt/homebrew/opt/colima/bin/colima" => "gusto-lima"
+
+    gusto_profile_config = <<~YAML
+      cpu: 4
+      disk: 60
+      memory: 8
+      arch: host
+      runtime: docker
+      hostname: ""
+      kubernetes:
+      enabled: false
+      version: v1.24.3+k3s1
+      k3sArgs:
+        - --disable=traefik
+      autoActivate: true
+      network:
+      address: false
+      dnsHosts:
+        host.docker.internal: host.lima.internal
+      forwardAgent: false
+      docker: {}
+      vmType: vz
+      rosetta: false
+      mountType: virtiofs
+      mountInotify: false
+      cpuType: host
+      provision:
+        - mode: system
+          script: sysctl -w vm.max_map_count=262144
+      sshConfig: true
+      mounts: []
+      env: {}
+    YAML
+
+    colima_profile_path = Pathname.new(Dir.home)/".colima/gusto"
+    colima_profile_path.mkpath
+    File.write(colima_profile_path/"colima.yaml", gusto_profile_config)
```

Comment:
Does this work? I am pretty sure that there are sandbox things for homebrew for writing to home directory.

Reply:
@technicalpickles Good catch! It looks like chef  created the file when testing if this logic will create the file. Turns out hyou cannot create directories in in the user home directory in Formulas https://github.com/orgs/Homebrew/discussions/778 Removing this logic since we create this file in chef